### DISCUSSION TITLE
[FEATURE] Playwright - Allure Step integration with grouping in TraceViewer

### DISCUSSION THREAD

**Discussion by EldoradoEU** [allure-framework/allure2#2175] [Ideas & Feedback]
Hi,

Is it possible to implement something similar to test.step() in Java? I'm searching something to group actions in TraceViewer.
JS/TS version use Playwright test.step() in Allure reports which are responsible for grouping in TraceViewer.

**Comment by baev** [comment]
This is a request to support *Playwright Java*.

I'll be happy to review the PR 😉

**Reply by EldoradoEU** [reply]
https://github.com/microsoft/playwright-java/issues/1416

I also created feature proposition in Playwright Java.
